CactusTutorial.ipynb

```diff
@@ -434,26 +435,11 @@
     "TOVSolver::TOV_Gamma          = 2\n",
     "TOVSolver::TOV_K              = 100\n",
     "\n",
-    "# Set equation of state for evolution\n",
-    "EOS_Omni::poly_gamma                   = 2\n",
-    "EOS_Omni::poly_k                       = 100\n",
-    "EOS_Omni::gl_gamma                     = 2\n",
-    "EOS_Omni::gl_k                         = 100\n",
-    "\n",
```

Comment:
I may actually opt to add back the 

"# Set equation of state for evolution\n",
"EOS_Omni::poly_gamma                   = 2\n",
"EOS_Omni::poly_k                       = 100\n",
"EOS_Omni::gl_gamma                     = 2\n",
"EOS_Omni::gl_k                         = 100\n",

options to both this and the gallery parfile (even if they just set
things to the same value as the defaults) so that these sample parfiles
do not rely on the magic of default values for parameters.

Reply:
Agreed

Reply:
done.